core/src/main/java/hudson/model/Job.java

```diff
@@ -72,6 +72,8 @@
 import net.sf.json.JSONObject;
 
 import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.StringUtils;
+import static org.apache.commons.lang.StringUtils.trim;
```

Comment:
static imports usually imported after all imports. Do you manage it manually? Usually all java IDEs handling this.